pkg/jira/client.go

```diff
@@ -93,14 +96,22 @@ func (e Errors) String() string {
 // Header is a key, value pair for request headers.
 type Header map[string]string
 
+// MTLS authtype specific config.
+type MTLSConfig struct {
+	CaCert     string
+	ClientCert string
+	ClientKey  string
+}
+
 // Config is a jira config.
 type Config struct {
-	Server   string
-	Login    string
-	APIToken string
-	AuthType AuthType
-	Insecure *bool
-	Debug    bool
+	Server     string
+	Login      string
+	APIToken   string
+	AuthType   AuthType
+	Insecure   *bool
+	Debug      bool
+	MTLSConfig MTLSConfig
```

Comment:
Wasn't sure if this should be a pointer instead?

Reply:
Should be ok even without pointer